radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<h3>Pros</h3>
+<ul>
+<li>Provides incredible integration and tooling</li>
+<li>Well structured pull requests for code reviews</li>
+<li>Ability to integrate bots to accomplish tasks</li>
+<li>Builtin security scanning</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Occasional outages can affect engineering</li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Clair,Infrastructure,Invest,leaning,"<p><a href=""https://coreos.com/clair/docs/latest/"">Clair</a> is a security scanner for Docker containers. Using Clair you are able to determine if there are known security issues with your container.</p>
+<h3>Pros</h3>
+<ul>
+<li>Integrates well with a private Docker registry</li>
+<li>Free</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>No UI</li>
+<li>Configuration isn’t straightforward</li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Jira,Infrastructure,Invest,lock,"<p><a href=""https://www.atlassian.com/software/jira"">Jira</a> is project management software.</p>
+<h3>Pros</h3>
+<ul>
+<li>Can use multiple workflows to accommodate different agile practices</li>
+<li>Not the easiest interface to work with’</li>
+<li>Cloud hosted</li>
+<li>Good plugin support</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Can feel really clunky at times</li>
+<li>Teams can be blocked when there are service interruptions</li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+SonarQube,Infrastructure,Invest,confident,"<p><a href=""https://www.sonarqube.org"">SonarQube</a> is a platform for continuous inspection of code quality to perform automatic reviews with static analysis of code to detect bugs, code smells, and security vulnerabilities.</p>
+<h3>Pros</h3>
+<ul>
+<li>Easy way to integrate code quality metrics with CI/CD pipeline</li>
+<li>Good resource for improving code quality during the writing process from within the IDE</li>
+<li>Supports most common languages and IDEs</li>
+<li>Allows high level monitoring of code quality across many projects quickly</li>
+<li>Detects code smells and evaluates technical debt</li>
+<li>Linting tool is free</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>The Free Qube version of the product only supports scanning master branch</li>
+<li>Cloud based version can be costly</li>
+<li>Limited</li>
+</ul>
+<p>When decision made: 9/29/2020</p>
+<p>When decision will be revisited: 9/22/2022</p>
+"
+Prometheus + Grafana,Observability,Invest,confident,"<p><a href=""https://prometheus.io/"">Prometheus</a> is an open source time series database. Prometheus implements a pull model for metrics rather than a push model. Meaning your applications need to have an endpoint, typically <code>/metrics</code> that Prometheus can scrape metrics from.</p>
```

Comment:
I thought we were moving away from this to relying more on New Relic

Reply:
Downgraded to divest